ds/data/processed/hebci/ce_cpd_income.lep.yaml

```diff
@@ -0,0 +1,31 @@
+api_doc_url: https://www.hesa.ac.uk/data-and-analysis/providers/business-community/
+api_type: FETCH
+data_date: 20200928 
+description_long: Income from continuing professional development and continuing education.
+description_short: Continuing professional development and education income
+description: Income from continuing professional development and continuing education.
+endpoint_url: https://www.hesa.ac.uk/data-and-analysis/providers/business-community/table-2b.csv
+framework_group: knowledge_exchange
+is_experimental: False
+is_public: True 
+order: [year, lep_id, lep_year_spec, value.id] 
+region:
+  type: LepRegion
+  source_url: LepRegion.source_url 
+  source: LepRegion.source 
+schema:
+  lep_id: 
+    type: LepRegion.lep_id 
+  lep_year_spec: 
+    type: LepRegion.lep_year_spec 
+  value:
+    description: Continuing professional development and education income
+    id: ce_cpd_income
+    label: Income
+    type: GBP
```

Comment:
(3) we need to add `data_type: int` to all currencies because of how the average gets shown. It might be good to add this in the spec as a comment to avoid further confusion.